org-roam-graph.el

```diff
@@ -144,120 +141,109 @@ into a digraph."
     (let* ((nodes (org-roam-db-query node-query))
            (edges-query
             `[:with selected :as [:select [file] :from ,node-query]
-              :select :distinct [to from] :from links
-              :where (and (in to selected) (in from selected))])
+                    :select :distinct [to from] :from links
+                    :where (and (in to selected) (in from selected))])
```

Comment:
This was due to `emacsql-fix-vector-indentation`:
```
When called, advise ‘calculate-lisp-indent’ to stop indenting vectors.
Once activate, vector contents no longer indent like lists.
```
I'm not sure which indentation style we want to keep. The one with `emacsql-fix-vector-indentation` seems a little cleaner, but then all contributors would be required to call that function before making changes to these queries. :man_shrugging:

Reply:
Good to know. I wasn't aware that was being used. I agree with both points (that's a bit cleaner looking, but also a hurdle for potential contributors).  One possibility is  to add per-file before-save-hook functions to ensure it is run. Though, I'm not too fond of evaling code when opening a file.